app/views/admin/work_reports/new.html.erb

```diff
@@ -71,7 +71,7 @@
                                   deposited
                                   version_draft
                                   decommissioned ]%>
-          <%= form.collection_check_boxes(:state, states_in_order.map { |state| [state.to_s, I18n.t(state, scope: 'work.state')] }, :first, :last) do |b| %>
+          <%= form.collection_check_boxes(:state, states_in_order.map { |state| [state.to_s, I18n.t(state, scope: 'work.state')] }, :first, :last, include_hidden: false) do |b| %>
```

Comment:
If you make this change, then https://github.com/sul-dlss/happy-heron/blob/main/app/services/admin/work_report_query.rb#L35 should also be changed. that line was stripping out the empty one.